docs/source/prometheus.rst

```diff
@@ -40,6 +40,13 @@ dask_scheduler_tasks_forgotten_total
        Task groups on the scheduler which have all tasks in the forgotten state are not
        included.
 
+dask_scheduler_tasks_compute_seconds_total
+    Total time (per prefix) spent computing tasks
+dask_scheduler_tasks_transfer_seconds_total
+    Total time (per prefix) spent transferring
+dask_scheduler_tasks_output_bytes_total
```

Comment:
I changed the labels.
This makes me think that it could be valuable to measure the _input_ bytes as well, so that we can get information about compression / expansion points in the graph.